src/puppetlabs/puppetdb/cli/time_shift_export.clj

```diff
@@ -187,14 +225,16 @@
                       (now))]
     {:input (:input options)
      :output (:output options)
-     :shift-to-time parsed-time}))
+     :shift-to-time parsed-time
+     :format (:format options)}))
 
 (defn- validate-cli!
   [args]
   (let [specs [["-i" "--input ARCHIVE" "Path to a .tgz file containing the export to be read."]
                ["-o" "--output ARCHIVE" "Path to where the time shifted .tgz file will be saved."
                 :default nil]
-               ["-t" "--shift-to-time DATE" "Reference DATE in UTC format to which all the timestamps will be shifted."]]
+               ["-t" "--shift-to-time DATE" "Reference DATE in UTC format to which all the timestamps will be shifted."]
+               ["-f" "--format stockpile" "When used on archive export, filenames will be renamed to stockpile format"]]
```

Comment:
I think that "Transforms the time shifted .tgz file into stockpile format." is more succinct and in line with the rest of the usage details.